sentence_transformers/SentenceTransformer.py

```diff
@@ -281,8 +281,8 @@ def encode(
         :param normalize_embeddings: Whether to normalize returned vectors to have length 1. In that case,
             the faster dot-product (util.dot_score) instead of cosine similarity can be used.
 
-        :return: By default, a list of numpy vectors is returned. If convert_to_tensor, a stacked tensor is returned.
-            If convert_to_numpy, a list of numpy vectors is returned.
+        :return: By default, a numpy array is returned. If convert_to_tensor, a stacked tensor is returned.
+            If convert_to_numpy, a numpy array is returned.
```

Comment:
```suggestion
        :return: By default, a 2d numpy array with shape [num_inputs, output_dimension] is returned. If only one string
            input is provided, then the output is a 1d array with shape [output_dimension]. If `convert_to_tensor`, a
            torch Tensor is returned instead.
```
Perhaps we can be more explicit and also list the output shape, like this?

Reply:
I'm also more in favor of what you're suggesting. I just didn't want to make too big changes as it's only my second contribution.

Reply:
All good! I appreciate you spotting this & your assistance.